_posts/2016/09/2016-09-01-election-procedures.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+authors: ["Kate Hertweck"]
+title: "Election Announcement: Amending Steering Committee election procedures"
+date: 2016-09-01
+time: "00:10:00"
+category: ["steering committee"]
+---
+
+The Steering Committee will be holding a special election on October 10-14
+regarding the following amendment to Steering Committee elections:
+
+**"Following an election, the new Steering Committee will meet jointly 
+with the previous Steering Committee for no less than 60 days. During 
+these joint meetings, the new SC will not have voting privileges. 
+After 30 days, voting privileges are transferred to the new SC."**
+
+We believe this amendment is necessary to provide continuity in leadership 
+following elections as the Steering Committee transitions to its new members. 
+We envision the timeline as follows:
+
+* November: Elections announced (90 days prior)
+* January: Candidate applications due; lab meeting to discuss candidates
+* February: Elections, first joint meeting, new committee nominates officers
+* March: Second joint meeting, new committee elects officers
```

Comment:
Can we expand this?

- March: Second joint meeting, new committee elects officers (as first motion on the agenda), old committee attend to answer questions since they no longer has voting rights.